README.md

```diff
@@ -18,11 +15,11 @@
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+**I used HTML to implement the Hello World Project, implemented the GitHub flow, and adopted best practices in the process of documenting the project.**
 
 
-To get a local copy up and running follow these simple example steps.
+## To request a local copy of my project, please follow these steps
+Request this url: https://github.com/ogagaoghene/hello-world in your web browser. Once opened navigate to the top left level of the project a green code download button will be visible on the righthand side. Select download Zip option from drop down menu. Once the download is complete you will be able to access my project locally.
 
 ### Prerequisites
```

Comment:
[OPTIONAL]  I kindly suggest you to remove this section because you don't need it in this project.